#input  
int64 objectId
bool isPick
bool isTeleop
float64 heightOfObject
---
#output
float64 heightOfObject
bool success
---
#feedback
int64 curState
string curStateInfo